CLAUDE.md

```
@@ -89,6 +89,65 @@ Claude-mem is designed with a clean separation between open-source core function
+FIX_FILES=$(git diff --name-only HEAD~N..HEAD -- '*.ts' '*.tsx' | sort)  # N = number of fix commits
+UNCOVERED=$(comm -23 <(echo "$FEATURE_FILES") <(echo "$FIX_FILES"))
+echo "Uncovered files: $(echo "$UNCOVERED" | wc -l)"
+echo "$UNCOVERED"
+# Review each uncovered file manually or via: cr review --base main --files $UNCOVERED
+
+# 3. E2E validation suite (80/80 gate)
+bash .local/tests/e2e/multi-node-validation.sh
+```
+
+**During staging**, run a full-diff sweep periodically (every ~4 rounds) instead of only at the end:
+```bash
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)